vertx-web/src/main/java/io/vertx/ext/web/impl/HttpServerRequestWrapper.java

```diff
@@ -387,6 +391,11 @@ public HttpServerRequest routed(String route) {
     return this;
   }
 
+  @Override
+  public Context context() {
+    return (Context) context;
```

Comment:
that will not work

Reply:
it should provide simple `request.context()` we are not takling of the same kind of context